### DISCUSSION TITLE
A question about the looptime

### DISCUSSION THREAD

**Discussion by LNTU-YS** [groops-devs/groops#276] [Q&A]
Hello,

I have encountered a problem like this. When I use “SlrParametrization ”and “StaticPositions”, I use one day's data to obtain station coordinates. What should I do if I want to use multiple days' data to determine station coordinates now? I tried the time loop and found that only the data from the first day is used each time. My settings are shown in the following figure:

![image](https://github.com/user-attachments/assets/4ecbb6c5-2230-473c-a36c-b2f12f7ba607)

I have also tried “TimeSeries -Monthly ”and “TimeSeries -EveryMonth ”, but found that they cannot achieve the results I want. I don't know where the problem lies, I hope you can give me some ideas.

With Kind Regards
YUAN

**Comment by akvas** [comment]
Hi @LNTU-YS,

I think setting sampling to 1 (= 1 day in MJD) should solve this problem. This will loop from January 1st up to and including January 31st.

Cheers,
Andreas

**Reply by LNTU-YS** [reply]
Hello Andreas,
﻿
Thank you very much for your reply and guidance, but it may be that I didn't describe it clearly and I don't want to do daily calculations.I would like to try if it is possible to use 31 days of data for simultaneous calculations.
﻿
With Kind Regards
YUAN

**Comment by akvas** [comment]
Hi @LNTU-YS,

thanks for the clarification. LoopTime will not help you in that case, as it will process each day individually (i.e. just execute the program for each day). 

Without the .xml file, I can only guess but two things to check are:

- Do the input data files really contain a whole month of data
- Is the parametrisation (specifically all timeIntervals) set correctly to one month 

Let me know if that helps,
Andreas

**Reply by LNTU-YS** [reply]
Hello Andreas,
﻿
Regarding the two points you mentioned, I don't think the problem lies here. I set the time to one month and have sufficient data. In fact, I have successfully used it for daily calculations, but the situation I mentioned earlier occur when solving on multiple days.
﻿
With Kind Regards
YUAN
